content/english/security-fundamentals/cyber-security-cause-&-effect.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Cyber security: Cause & Effect"
+description: ""
+draft: false
+weight: 4
+---
+
+
+Cyber security in practice is more than just not falling for scams online. What exactly are the common risks that users online ignore and that hackers take advantage of?  
```

Comment:
Do we want to include examples of how one can enable 2FA and password management?

Reply:
That sounds good, I'll add that in